src/Microsoft.Health.Dicom.Core/Features/Query/Model/FilterConditions/QueryFilterCondition.cs

```diff
@@ -15,6 +15,8 @@ public QueryFilterCondition(DicomTag tag)
 
         public DicomTag DicomTag { get; }
 
+        public CustomTagFilterDetails CustomTagFilterDetails { get; set; }
```

Comment:
nullable?

Reply:
This implicitly allows it to be null I think.

---
In reply to: [582547364](https://github.com/microsoft/dicom-server/pull/567#discussion_r582547364) [](ancestors = 582547364)